sys/net/gnrc/network_layer/ipv6/ext/rh/gnrc_ipv6_ext_rh.c

```diff
@@ -10,29 +11,91 @@
  * @{
  *
  * @file
+ * @author Cenk Gündoğan <cnkgndgn@gmail.com>
+ * @author Martine Lenders <m.lenders@fu-berlin.de>
  */
 
-#include <stdbool.h>
+#include "net/ipv6/ext/rh.h"
+#include "net/gnrc.h"
 
-#include "net/protnum.h"
-#include "net/ipv6/ext.h"
-#include "net/gnrc/ipv6/ext/rh.h"
+
```

Comment:
extra line here

Reply:
```
vera ./sys/net/gnrc/network_layer/ipv6/ext/rh/gnrc_ipv6_ext_rh.c
./sys/net/gnrc/network_layer/ipv6/ext/rh/gnrc_ipv6_ext_rh.c:21: too many consecutive empty lines
```

Reply:
Intentional to set off the the `#ifdef` ;-)

Reply:
> ```
> vera ./sys/net/gnrc/network_layer/ipv6/ext/rh/gnrc_ipv6_ext_rh.c
> ./sys/net/gnrc/network_layer/ipv6/ext/rh/gnrc_ipv6_ext_rh.c:21: too many consecutive empty lines
> ```

I guess because it is not recognizing the `#ifdef`? I disagree with `vera++` here.

Reply:
If not for the `#ifdef` I probably still would set it off, since this is one of the RH implementing libraries.

Reply:
Oh I only now see the extra extra newline. Somehow GitHub on my other machine showed it to me differently ^^" Will fix!

Reply:
Fixed and squashed